repl.py

```diff
@@ -365,11 +369,16 @@ def repl(symbols):
 def print_usage(prog_name, out):
     out.write("Usage: {} [options...] <auth.json>\n")
     out.write("\n")
-    out.write("    <auth-json>: See ReadMe.md for information on how to create this file.\n")
```

Comment:
Looks like I wrote `auth.json` in most places but wrote `auth-json` in this one spot.  Whoops.  Should probably switch to one or the other.